README.adoc

```diff
@@ -358,12 +366,12 @@ which can be used for span creation and customization, including setting their t
 
 == Testing the services
 
-A few tests have been included for you to test the basic functionality of the services. If a test
-failure occurs, then you must have introduced a bug into the code. These tests will run automatically
-as a part of the Maven build process (when you run `mvn install`). You can also run these tests separately
-from the build using the `mvn verify` command, but make sure that the servers are stopped beforehand.
+A few tests are included for you to test the basic functionality of the services. If a test
+failure occurs, then you might have introduced a bug into the code. These tests will run automatically
+as a part of the Maven build process when you run `mvn install`. You can also run these tests separately
```

Comment:
I'd say, "...run **the** `mvn install` **command**." It sounds kind of obvious/redundant, but we might want it later if we ever translate these guides. :)

Reply:
Ok :) Also added this for the openapi guide